nixos/modules/services/networking/shairport-sync.nix

```diff
@@ -6,6 +6,37 @@ let
 
   cfg = config.services.shairport-sync;
 
+  # This is based off the upstream versions, but they can't be used directly because
```

Comment:
Could we easily patch it or would that complicate the build to much?

Reply:
Do you think you can fix the double prefix issue at its core?

> Patching them is probably a simple s/shairport-sync/${user}/g, but I haven't checked.

That would be really simple.

> Another option is to simply remove the ability to set the user/group, and force it to shairport-sync so the upstream policies can be used directly.

I am not sure what other people think but if upstream doesn't support it either, that might make things just way easier.

Reply:
> How's this?
> 
> [vs49688@20bd259](https://github.com/vs49688/nixpkgs/commit/20bd259c782a1126b8aa8bb089e6d5fc1dbf25de)

Nice. Could you update the PR?